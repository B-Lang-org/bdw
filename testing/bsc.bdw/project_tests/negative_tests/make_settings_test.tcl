

# Sets the makefile and appropriate targets
handlers::set_make_options Makefile

# Opens the specified project
handlers::open_project project.bspec

# Sets the makefile and appropriate targets
handlers::set_make_options Makefile

# Sets the compilation type to make
handlers::set_compilation_type make

# Sets the make options with no argument
handlers::set_make_options

# Sets the make options with wrong argument
handlers::set_make_options wrong_file

# Sets the make options with wrong argument
handlers::set_make_options negative_tests.exp

# Sets the makefile without an argument fot -target option
handlers::set_make_options Makefile -fullclean

# Sets the makefile with wrong options
handlers::set_make_options Makefile -targett run

# Sets the makefile and appropriate targets
handlers::set_make_options Makefile -target run -target run -clean clean

# Closes the project
handlers::close_project

# Exits the workstation
exit
