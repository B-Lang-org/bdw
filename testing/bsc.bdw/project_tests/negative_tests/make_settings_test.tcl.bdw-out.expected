There is no open project
The compilation type is not set to "make"  for the current project.
The Makefile is not specified
The file wrong_file doesn't exist.
Argument -fullclean expects an argument none found
Unknown option: -targett
The option -target has already been specified
